developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,34 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+[[Pry]]
+== Debugging Foreman with Pry
+https://github.com/pry/pry[Pry] is a runtime developer console and IRB (interactive Ruby) alternative with powerful introspection capabilities.
+You can use use Pry as a developer console or as a debugger.
+Pry gem is required by Foreman, meaning that Bundler installs it for you.
+
+To invoke the debugger, place `binding.pry` somewhere in your code as follows:
+[source, ruby]
+....
+require 'pry'
+binding.pry
```

Comment:
could you make it a one-liner as @ekohl suggested ?
I think it will be easier for newcomers

```suggestion
require 'pry'; binding.pry
```

Reply:
Yes of course!